src/mono/mono/metadata/metadata-update.c

```diff
@@ -423,6 +423,17 @@ mono_image_open_dmeta_from_data (MonoImage *base_image, uint32_t generation, gco
 	return dmeta_image;
 }
 
+static gpointer
+open_dil_data (MonoImage *base_image G_GNUC_UNUSED, gconstpointer dil_src, uint32_t dil_length)
+{
+	/* TODO: find a better memory manager.  But this way we at least won't lose the IL data. */
+	MonoMemoryManager *mem_manager = (MonoMemoryManager *)mono_alc_get_default ()->memory_manager;
```

Comment:
I know I could use `m_image_get_memory_manager (base_image)` here, but that function looks different in 6.0-preview4, so for the sake of the backport, I'd rather dump the delta IL into the default memory manager for now.